static/app/views/settings/project/projectOwnership/index.tsx

```diff
@@ -284,6 +315,15 @@ tags.sku_class:enterprise #enterprise`;
                       >
                         {t('Add CODEOWNERS File')}
                       </CodeOwnerButton>
+                    ) : (
+                      <CodeOwnerButton
+                        onClick={this.handleAddCodeOwnerRequest}
+                        size="small"
+                        priority="primary"
+                        data-test-id="add-codeowner-request-button"
+                      >
+                        {t('Request to Add CODEOWNERS File')}
```

Comment:
@NisanthanNanthakumar in SaaS, will members see two different buttons to request someone to to set up code owners?

Reply:
@scefali yea that was in the original design